common/classes/com/arjuna/common/logging/commonI18NLogger.java

```diff
@@ -51,17 +52,18 @@ public interface commonI18NLogger {
     public void warn_common_ClassloadingUtility_3(String arg0, String arg1, @Cause() Throwable arg2);
 
     @Message(id = 48006, value = "cannot create new instance of {0}", format = MESSAGE_FORMAT)
-    @LogMessage(level = WARN)
-    public void warn_common_ClassloadingUtility_4(String arg0, @Cause() Throwable arg1);
+    public String warn_common_ClassloadingUtility_4(String arg0, @Cause() Throwable arg1);
 
     @Message(id = 48007, value = "cannot access {0}", format = MESSAGE_FORMAT)
-    @LogMessage(level = WARN)
-    public void warn_common_ClassloadingUtility_5(String arg0, @Cause() Throwable arg1);
+    public String warn_common_ClassloadingUtility_5(String arg0, @Cause() Throwable arg1);
 
     @Message(id = 48008, value = "cannot initialize from string {0}", format = MESSAGE_FORMAT)
     @LogMessage(level = WARN)
     public void warn_common_ClassloadingUtility_6(String arg0, @Cause() Throwable arg1);
 
+    @Message(id = 48009, value = "cannot create new instance of {0}", format = MESSAGE_FORMAT)
+    @LogMessage(level = FATAL)
+    public void fatal_common_ClassloadingUtility_7(String arg0, @Cause() Throwable arg1);
```

Comment:
Where do you use this FATAL log message?

Reply:
I don't so I will remove it.